static/src/javascripts/projects/common/modules/lazyload.js

```diff
@@ -43,9 +44,13 @@ define([
                         crossOrigin: true
                     }).then(
                         function (resp) {
-                            into.html(opts.beforeInsert(resp.html));
-                            into.addClass('lazyloaded');
-                            opts.success(resp);
+                            var componentHtml = opts.beforeInsert(resp.html);
+                            fastdom.write(function () {
+                                into.html(componentHtml);
+                                into.addClass('lazyloaded');
+                                opts.success(resp);
```

Comment:
As before, depends what this triggers. I'm not sure whether mediator itself automatically defers running code that subscribes to events you publish or whether you need to explicitly do it yourself. I think it's probably the latter though.